vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -373,6 +373,28 @@ public boolean isInvalid() {
         return isInvalidBoolean();
     }
 
+    /**
+     * Set to <code>false</code> to hide the clear button which clears the datepicker.
```

Comment:
It's false by default, right?
Also, could state that the clear button is shown when the component has a value. (see my PR for combo box https://github.com/vaadin/vaadin-combo-box-flow/pull/247/files)

Reply:
Yeap, it's false. Changed the wording. Added demo.